sdk/core/core/src/az_http_policy.c

```diff
@@ -30,6 +28,14 @@ AZ_NODISCARD AZ_INLINE az_result az_http_pipeline_nextpolicy(
       &(p_policies[1]), p_policies[0]._internal.p_options, p_request, p_response);
 }
 
+static AZ_NODISCARD az_result _az_http_pipeline_nextpolicy(
```

Comment:
inline

Reply:
I removed it in the next iteration, but the whole goal of this specific helper was to generate static body for the inline function, so its address can be taken. As you can se all this function was doing - was to call the inline version. At the same time we didn't want to remove inline status from the az_http_pipeline_nextpolicy(), because that could potentially increase the stack size for no real reason, since pipelines do heavily rellay on a deep chain of nested calls of functions.